src/modules/tools/spindle/SoftSerial/SoftSerial.h

```diff
@@ -13,7 +13,7 @@
 /** A software serial implementation
  *
  */
-class SoftSerial: public Stream {

```

Comment:
This shouldn't be necessary as it is no tin the main branch you simply needed to merge with the current edge branch. I think this may well conflict.

Reply:
yeah sorry, my bad....
new to this github thing ;)